### DISCUSSION TITLE
Convert relative links outside of /docs to use repo_url

### DISCUSSION THREAD

**Discussion by mauibar** [mkdocs/mkdocs#3812] [Ideas]
Hello,

I have md files that contains relative links to files within my repo that are outside of `/docs`.
For instance:

```
- Pre-built GitHub workflows
  - [CI](../.github/workflows/ci.yaml):
```

I would have liked to keep the relative links for Github and wondered if there was a feature/plugin for mkdocs that could convert those relative links outside of `/docs` to use `repo_url` and `edit_uri`  as base url when on the mkdocs page?

Thanks

**Comment by pawamoy** [comment]
Check out the [catalog](https://github.com/mkdocs/catalog/) :smile:! I'll move this to discussions :slightly_smiling_face:

**Comment by mauibar** [comment]
Thanks checked it out, unfortunately didn't find what I needed

**Comment by githoober** [comment]
@pawamoy are you saying it's already addressed in some way and we need to find that implementation in the catalog?

**Reply by pawamoy** [reply]
I'm not saying it's addressed, nor that it's not addressed, I'm simply answering the initial question:

> I [...] wondered if there was a feature/plugin for mkdocs that could convert those relative links outside of /docs to use repo_url and edit_uri as base url when on the mkdocs page?

My answer being, I don't know, check out the catalog! :slightly_smiling_face:

**Reply by githoober** [reply]
I got it. Unfortunately, there is nothing in the catalog that provides the required functionality.

**Reply by pawamoy** [reply]
Feel free to open a feature request then, or initiate some discussion on how to solve that :)
